src/core/server/saved_objects/migrations/actions/integration_tests/actions.test.ts

```diff
@@ -425,6 +425,10 @@ describe('migration actions', () => {
   describe('cloneIndex', () => {
     afterAll(async () => {
       try {
+        // Restore the default setting of 1000 shards per node
+        await client.cluster.putSettings({
+          persistent: { cluster: { max_shards_per_node: null } },
+        });
```

Comment:
Maybe we should put that in a `afterEach` block instead of `afterAll`? That would avoid the tests to be forced to run in a specific order (e.g if we later add a new test after the one introduced in the PR)

Reply:
good point, :+1: